projects/storefrontapp-e2e-cypress/cypress.config.ts

```diff
@@ -16,14 +16,16 @@ export default defineConfig({
   env: {
     CLIENT_ID: 'mobile_android',
     CLIENT_SECRET: 'secret',
-    API_URL: 'https://40.76.109.9:9002',
+    API_URL:
+      'https://api.cp96avkh5f-integrati2-s1-public.model-t.cc.commerce.ondemand.com',
```

Comment:
Can we do not commit any changes in this file? Just let's keep them locally.

Reply:
Changed